src/uniparc/adapters/uniParcConverter.ts

```diff
@@ -4,15 +4,14 @@ import { EntryType } from '../../uniprotkb/adapters/uniProtkbConverter';
 
 import EntrySection from '../types/entrySection';
 
-export enum XRefsInternalDatabases {
-  Reviewed = 'UniProtKB/Swiss-Prot',
-  Unreviewed = 'UniProtKB/TrEMBL',
-}
+export type XRefsInternalDatabases =
+  | 'UniProtKB/Swiss-Prot'
+  | 'UniProtKB/TrEMBL';
 
-export const databaseToEntryType: Record<string, EntryType | undefined> = {
-  [XRefsInternalDatabases.Reviewed]: EntryType.REVIEWED,
-  [XRefsInternalDatabases.Unreviewed]: EntryType.UNREVIEWED,
-};
+export const databaseToEntryType = new Map<string, EntryType>([
```

Comment:
```suggestion
export const databaseToEntryType = new Map< XRefsInternalDatabases, EntryType>([
```

Reply:
Actually reverting and putting an union. It's because I'm not necessary doing `.get()` with one of these two options, but rather with any database name, so I need the map to accept any string, but then I have to take into account that the result of `get()` will be either an `EntryType`, or undefined (for any database which is not an `XRefsInternalDatabases`)